bson/mgocompat/registry.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package mgocompat
```

Comment:
As I mentioned in Slack, I don't think it's a good idea for this file to exist right now. It might confuse users reading our docs because they may expect `mgocompat.NewRegistryBuilder().Build()` to return a mgo-compatible registry, which won't be the case until all of the codecs are completed. Is it possible to move this into a test file instead so it's accessible by our tests? As we do codecs, we can update the test file and then just rename it without the `_test` suffix when we do the registry ticket.

Reply:
I made the builder and the registry private